### PR TITLE
Fixing Mandiant ASM Issue Status Sync

### PR DESCRIPTION
**Pull request by chrishultin**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fixes an issue where ASM issue status was not synced to Cortex XSOAR.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MandiantAdvantageAttackSurfaceManagement/ReleaseNotes/1_0_5.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Mandiant Attack Surface Management
+
+- Fixing issue where closing an incident in Mandiant Advantage ASM did not close the corresponding incident in Cortex XSOAR.
```
Comment (YuvHayun) [inline Packs/MandiantAdvantageAttackSurfaceManagement/ReleaseNotes/1_0_5.md:6]:
```suggestion
- Fixed an issue where closing an incident in Mandiant Advantage ASM did not close the corresponding incident in Cortex XSOAR.
```

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Looks good!
I left one small comment regarding the release notes.
Please attach a short demo video with the following sequence:
- An incident being fetched (show the incident in the incidents tab and the incident status)
- Closing that incident in Mandiant Advantage ASM UI.
- Show that the fetched incident in xsoar is closed: both the status in the context and the closed updated in the incident page.

**Comment by YuvHayun** [comment]
Hey @chrishultin how is it going?
Let me know if you need any help with the changes.

**Comment by chrishultin** [comment]
> Hey @chrishultin how is it going? Let me know if you need any help with the changes.

Sorry for the delay.  I've tweaked the release notes as per your request, and sent you a video via Slack of the system working.

**Comment by YuvHayun** [comment]
Merging to internal pr.
Thx for the fix!

**Review by YuvHayun — APPROVED** [review]

**Pull request merged**